packs/spells/ulcerous-canker.json

```diff
@@ -37,7 +37,7 @@
             }
         },
         "description": {
-            "value": "<p>Weeping sores open on the target as you expose it to decay, dealing 1d4 void damage and 1d4 poison damage plus @Damage[(ceil(@item.rank / 2))[bleed]]{persistent bleed} damage. The target must attempt a Fortitude save.</p><hr /><p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target takes half the initial damage and no persistent bleed damage.</p>\n<p><strong>Failure</strong> The target takes full damage.</p>\n<p><strong>Critical Failure</strong> The creature takes double initial damage and double persistent bleed damage.</p><hr /><p><strong>Heightened (+2)</strong> The void damage and poison damage each increase</p>"
+            "value": "<p>Weeping sores open on the target as you expose it to decay, dealing 1d4 void damage and 1d4 poison damage plus @Damage[(ceil(@item.rank / 2))[bleed]]{persistent bleed} damage. The target must attempt a Fortitude save.</p><hr /><p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target takes half the initial damage and no persistent bleed damage.</p>\n<p><strong>Failure</strong> The target takes full damage.</p>\n<p><strong>Critical Failure</strong> The creature takes double initial damage and double persistent bleed damage.</p><hr /><p><strong>Heightened (+2)</strong> The void damage and poison damage each increase by 1d4, and the persistent bleed damage increases by 2.</p>"
```

Comment:
I don't believe the persistent damage formula is correct.

Reply:
You are right, it wasn't.
It would be nice if there was a standardized way of writing it (something like `baseValue + heighteningMult*floor((@item.rank -@item.baseRank)/heighteningStep)`)